cmd/zpool/os/linux/zpool_vdev_os.c

```diff
@@ -239,7 +239,7 @@ check_sector_size_database(char *path, int *sector_size)
 	io_hdr.dxferp = inq_buff;
 	io_hdr.cmdp = inq_cmd_blk;
 	io_hdr.sbp = sense_buffer;
-	io_hdr.timeout = 10;		/* 10 milliseconds is ample time */
+	io_hdr.timeout = 10;		/* 10 milliseconds is sample time */
```

Comment:
ample is correct.

Reply:
Thanks a lot for the review! I'm going to upload reverts.